data/Ellsworth_2015/metadata.yml

```diff
@@ -0,0 +1,1225 @@
+    where thickness and area were measured on a subsample, whereas the remainder was
+    frozen and subsequently dried to a constant mass at 70°C. The leaf lamina dry
+    mass per unit area (Ma) was calculated from the ratio of dry mass to fresh area.
+    The dried sample was ground finely in a ball mill and used for analyses of total
+    N concentration, total P concentration, inorganic P (Pi) concentration, and starch
+    and soluble sugar concentrations. Leaf N concentration was analysed by elemental
+    analysis after combustion using a CHN elemental analyser (TruSpec micro; LECO
+    Corp., St. Joseph, MI, USA; or FLASH EA 1112 Series CHN analyser; Thermo-Finnigan,
+    Waltham, MA, USA). Leaf total P concentrations were measured after digesting dried
+    leaf tissue with concentrated sulfuric acid (H2SO4) and hydrogen peroxide (H2O2)
+    in a microwave digester apparatus (Berghof speedwave four; Berghof Products GmbH,
+    Eningen, Germany). The solutions containing total P or the Pi fraction were analysed
+    colorimetrically at 880nm (AQ2; SEAL Analytical, Ltd., Milwaukee, WI, USA) after
+    a standard molybdate reaction (Close & Beadle 2004). Analyses of N and P concentrations
+    used international standards run blind alongside the samples and are expressed
+    as N and P content (mmol m-2) in this manuscript due to differences in leaf thickness
+    among the species (Table1). Bulk leaf Pi was determined by extracting samples
+    in 0.3M trichloroacetic acid at 4°C before cold centrifuging at 9224xg (10000r.p.m.)
+    for 5min and collecting the filtrate (Close & Beadle 2004). The Pi concentrations
+    in the samples were determined against standards made with KH2PO4 in serial dilution.
+- var_in: LMA_g.m2
+  unit_in: g/m2
+  trait_name: leaf_mass_per_area
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
+  methods: All measurements were made on attached, intact leaves at the top of the
+    crown or the outer shell of the crown when open-grown, which meant accessing leaves
+    from 1 m up to 25 m high (Table 1). For tall species, access to the upper parts
+    of the tree crowns was achieved by three different means, an articulated boom
+    lift (Snorkel MHP13/35 Trailer Mounted Lift; Snorkel Ltd., Meadowbrook, Qld, Australia)
+    used at the Hawkesbury site in Richmond, NSW; a set of 36 m tall construction
+    cranes (Jaso crane J-4010; Jaso S.L., Idiazabal, Spain) at the nearby EucFACE
+    site in Richmond, NSW; and a custom-built steel-alloy canopy walkway going up
+    from ground level to 30 m height ('Illawarra fly') at Robertson, NSW. Canopy access
+    was not necessary at the Lesueur National Park site or at Davies Park, as trees
+    and shrubs were open-grown in each of the sclerophyll woodlands, and unshaded
+    leaves at the outside of the crown could be readily measured.
```

Comment:
In the readme file for this data, they say they measured LMA without petiole, so does that mean there should be a `measurement_remarks column` mentioning this?

Reply:
I'll add that.

Reply:
fixed